packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -149,6 +149,8 @@ class _FlutterPlatform extends PlatformPlugin {
     bool subprocessActive = false;
     bool controllerSinkClosed = false;
     try {
+      // Callback hookups' Futures are uninteresting.
+      // ignore: unawaited_futures
```

Comment:
Was trying to read the code to understand but I'm not sure either. It seems like you authored this part originally. Do you recall why?